packages/plugins/connection-manager/extension.ts

```diff
@@ -195,9 +195,15 @@ export default class ConnectionManagerPlugin implements SQLTools.ExtensionPlugin
   }
 
   // internal utils
-  private async _getTableName(node?: SidebarTable | SidebarView): Promise<string> {
+  private async _getTableName(node?: SidebarTableOrView): Promise<string> {
     if (node && node.value) {
       await this._setConnection(node.conn as ConnectionInterface);
+      switch(node.conn.dialect) {
+        case DatabaseDialect.PostgreSQL:
+          return [node.table.tableDatabase, node.table.tableSchema, node.table.name].join('.');
+        case DatabaseDialect.MySQL:
+          return [node.table.tableSchema, node.table.name].join('.');
```

Comment:
Also here, to make it query for the exact table name @jpincas was talking about.